index.js

```diff
@@ -0,0 +1,50 @@
+(function(global, f){
+
+  'use strict';
+
+  /*istanbul ignore next*/
+  if(module && typeof module.exports !== 'undefined'){
+    module.exports = f(
+      require('fluture'),
+      require('sanctuary-def'),
+      require('sanctuary-type-identifiers')
+    );
+  }else{
+    global.concurrify = f(
+      global.Fluture,
+      global.sanctuaryDef,
+      global.sanctuaryTypeIdentifiers
+    );
+  }
+
+}(/*istanbul ignore next*/(global || window || this), function(Future, $, type){
+
+  'use strict';
+
+  //  $$type :: String
+  var $$type = '@@type';
+
+  //  FutureType :: (Type, Type) -> Type
+  var FutureType = $.BinaryType(
+    type.parse(Future[$$type]).name,
+    'https://github.com/fluture-js/Fluture#readme',
+    Future.isFuture,
+    Future.extractLeft,
+    Future.extractRight
+  );
+
+  //  ConcurrentFutureType :: (Type, Type) -> Type
+  var ConcurrentFutureType = $.BinaryType(
+    type.parse(Future.Par[$$type]).name,
+    'https://github.com/fluture-js/Fluture#concurrentfuture',
```

Comment:
I would need to know the version which is used by the "peer". I could possibly do something like `require('fluture/package.json')`, but that won't work when people use this module via IIFE.